cli/src/psp-utils/buildCircom.ts

```diff
@@ -140,38 +146,15 @@ export async function generateCircuit({
     await sleep(10);
   }
 
-  fs.unlinkSync(path.join(sdkBuildCircuitDir, "verifyingkey.json"));
+  fs.unlinkSync(
+    path.join(sdkBuildCircuitDir, `verifyingkey${circuitName}.json`)
+  );
   fs.unlinkSync(path.join(sdkBuildCircuitDir, `${circuitName}_tmp.zkey`));
-  fs.unlinkSync(path.join(sdkBuildCircuitDir, `${circuitName}.r1cs`));
-  fs.unlinkSync(path.join(sdkBuildCircuitDir, `${circuitName}.sym`));
-}
-
-/**
- * Builds a barebone Circom + Anchor project given a circuit directory.
- * Initializes client-side typescript prover, on-chain groth16-solana verifier, builds the circom circuit, and compiles the anchor program.
- * @param circuitDir - The directory containing the circuit files.
- * @returns {Promise<void>}
- */
-export async function buildCircom(
-  circuitDir: string,
-  ptau: number,
-  programName: string
-) {
-  let circuitFileName = findFile({
-    directory: circuitDir,
-    extension: "circom",
-  });
-
-  console.log("🛠️️  Building circuit", circuitFileName);
-
-  await generateCircuit({
-    circuitFileName,
-    ptau,
-    programName,
-  });
-  console.log("✅ Circuit generated successfully");
-
-  console.log("🛠  Building on-chain program");
-  await executeAnchor({ args: ["build"] });
-  console.log("✅ Build finished successfully");
+  fs.unlinkSync(path.join(sdkBuildCircuitDir, `${circuitName}Main.r1cs`));
+  fs.unlinkSync(path.join(sdkBuildCircuitDir, `${circuitName}Main.sym`));
 }
+function camelToKebab(str: string): string {
+  return str
+    .replace(/([a-z0-9]|(?=[A-Z]))([A-Z])/g, '$1-$2')  // Match camelCasing and insert "-"
+    .toLowerCase();  // Convert the entire string to lowercase
+}
```

Comment:
Shouldn't it be in the [cli/src/psp-utils/utils.ts, close to the "toSnakeCase" function](https://github.com/Lightprotocol/light-protocol/blob/main/cli/src/psp-utils/utils.ts#L39)?

Reply:
good point, I think these functions are scattered, let me group these functions in zk.js.

Reply:
done